### PR TITLE
#16489: fix the redirect issue to new tab for tags and glossary

### LINKED ISSUE #16489: Unable to open glossary term in new tab from Table view
**Affected module**
UI

**Describe the bug**
When viewing a column's associated glossary terms, one may want to open the glossary term in a new tab. This is currently not possible with either middle-click or right-click + open in new tab

**To Reproduce**

image

**Expected behavior**
User should be able to open the glossary term in a new tab with either of the methods mentioned above

**Version:**
 - OpenMetadata version: 1.4

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes:

Fixes #16489

- I worked on fixing unable to open glossary term and tag in new tab
- Decrease mouseDelay of popover to 500ms on `Tags`

https://github.com/open-metadata/OpenMetadata/assets/66266464/76c804b1-bf63-4c7a-8649-42235b1a71db

#
### Type of change:

- [x] Bug fix
- [x] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

**Review by chirag-madlani — DISMISSED** [review]

File: openmetadata-ui/src/main/resources/ui/src/components/Explore/EntitySummaryPanel/SearchIndexSummary/SearchIndexSummary.test.tsx [hunk 1]
```diff
@@ -33,7 +33,9 @@ describe('SearchIndexSummary component tests', () => {
   it('Component should render properly, when loaded in the Explore page.', async () => {
     await act(async () => {
       render(
-        <SearchIndexSummary entityDetails={mockSearchIndexEntityDetails} />
+        <BrowserRouter>
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/Explore/EntitySummaryPanel/SearchIndexSummary/SearchIndexSummary.test.tsx:36]: We can have MemoryRouter as wrapper instead manually adding BrowserRouter check other specs for ref

File: openmetadata-ui/src/main/resources/ui/src/components/Explore/EntitySummaryPanel/SearchIndexSummary/SearchIndexSummary.test.tsx [hunk 2]
```diff
@@ -101,21 +103,23 @@ describe('SearchIndexSummary component tests', () => {
...
-                tagFQN: 'Tier.Tier1',
-                labelType: LabelType.Manual,
-                description: '',
-                source: TagSource.Classification,
-                state: State.Confirmed,
-              },
-            ],
-          }}
-        />
+        <BrowserRouter>
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/Explore/EntitySummaryPanel/SearchIndexSummary/SearchIndexSummary.test.tsx:106]: Same

File: openmetadata-ui/src/main/resources/ui/src/components/TagsInput/TagsInput.test.tsx [hunk 3]
```diff
@@ -39,12 +40,14 @@ describe('TagsInput', () => {
   it('should render TagsInput along with tagsViewer in version view', async () => {
     await act(async () => {
       render(
-        <TagsInput
-          isVersionView
-          editable={false}
-          tags={tags}
-          onTagsUpdate={mockOnTagsUpdate}
-        />
+        <BrowserRouter>
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/TagsInput/TagsInput.test.tsx:43]: same

File: openmetadata-ui/src/main/resources/ui/src/components/TagsInput/TagsInput.test.tsx [hunk 4]
```diff
@@ -59,11 +62,13 @@ describe('TagsInput', () => {
   it('should render tags container when not in in version view', async () => {
     await act(async () => {
       render(
-        <TagsInput
-          editable={false}
-          tags={tags}
-          onTagsUpdate={mockOnTagsUpdate}
-        />
+        <BrowserRouter>
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/TagsInput/TagsInput.test.tsx:65]: same

File: openmetadata-ui/src/main/resources/ui/src/components/TagsInput/TagsInput.test.tsx [hunk 5]
```diff
@@ -79,7 +84,9 @@ describe('TagsInput', () => {
   it('should render edit button when no editable', async () => {
     await act(async () => {
       render(
-        <TagsInput editable tags={tags} onTagsUpdate={mockOnTagsUpdate} />
+        <BrowserRouter>
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/TagsInput/TagsInput.test.tsx:87]: same

**Comment by Ashish8689** [comment]
@chirag-madlani  changes made as per comments!

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**